<launch>
  <param name="/use_sim_time" value="true" />
  <rosparam file="$(find lego_lol)/params/global.yaml" command="load" />

  <!-- <node pkg="nodelet" type="nodelet" name="lego_manager" args="manager" output="screen" />
  <node pkg="nodelet" type="nodelet" name="ImageProjection" args="load loam/ImageProjection lego_manager" output="log" />
  <node pkg="nodelet" type="nodelet" name="LaserOdometry" args="load loam/LaserOdometry lego_manager" output="log" /> -->

  <node pkg="alego" type="IP" name="IP" />

  <node pkg="alego" type="LO" name="LO" output="screen">
  </node>

  <node pkg="lego_lol" type="lol" name="lol" output="screen">
    <remap from="/corner" to="/corner_less" />
    <remap from="/surf" to="/surf_less" />
    <!-- <remap from="/outlier" to="/laser_cloud_outlier" /> -->
    <remap from="/odom" to="/odom/lidar" />
    <rosparam file="$(find lego_lol)/params/test.yaml" command="load" />
    <param name="fn_poses" value="$(find bag_file)/pcd/keypose_tf.pcd" />
    <param name="fn_corner" value="$(find bag_file)/pcd/corner_tf.pcd" />
    <param name="fn_surf" value="$(find bag_file)/pcd/surf_tf.pcd" />
    <param name="fn_outlier" value="$(find bag_file)/pcd/outlier_tf.pcd" />
  </node>
</launch>